shared-libs/user-management/src/users.js

```diff
@@ -770,9 +802,9 @@ const getUserSettings = async({ name }) => {
  */
 module.exports = {
   deleteUser: username => deleteUser(createID(username)),
-  getList: async () => {
-    const [ users, settings ] = await Promise.all([ getAllUsers(), getAllUserSettings() ]);
-    const facilities = await facility.list(users, settings);
+  getList: async (filters) => {
+    const [users, settings] = await getUsersAndSettings(filters);
+    const facilities = await facility.list(users);
     return mapUsers(users, settings, facilities);
   },
   getUserSettings,
```

Comment:
I believe we should also change how this function is implemented, because now it only picks the facility_id from the users doc, and the contact_id from the user-settings doc. I think we should get both from the user doc.